docs/Avoiding-Disruptions.md

```diff
@@ -0,0 +1,36 @@
+# Avoiding Disruptions
+
+Under certain conditions, MachineConfig updates will cause the Machine Config
```

Comment:
This should be rephrased as os updates aren't a machineconfig update.

Reply:
osImageURL is also a field in machineconfigs. I don't think we should differentiate there